ext/drsyms/elfutils/elf_memory.c.patch

```diff
@@ -1,11 +0,0 @@
-diff --git a/libelf/elf_memory.c b/libelf/elf_memory.c
```

Comment:
> we pin the elfutils commit to the next official release that contains the current commit (which is 0.191).

Isn't 0.192 the latest release?  But see the other comment: they reverted the fix we need so we'd need this patch again (assuming it works as-is still).

Reply:
Yes, but I was unsure if more patches will be needed to support that, so I decided to go with "closer" release. But I'm happy to update this MR to 0.192. Just let me know. Then we also don't have to remove and re-introduce the patch again.

Reply:
It's possible the patch as-is has problems with recent changes, or unrelated changes break something: hopefully our tests are enough to catch that.  Maybe worth a try with 0.192 and see if the tests succeed?

Reply:
I'll give it a try.

Reply:
Hmm... the update to 0.192 is not straight forward as the `elfutils/lib` component now contains sources which we need as well. I can have a look into that again, but for now I propose to just go with the 0.191 release.

Reply:
OK, 0.191 then.  Hopefully whoever updates to 0.192 will remember this patch and not have to debug SIGSEGV and re-discover it.

Reply:
As likely I will be the person who does this (within the next few weeks), the risk is low :)
This MR is ready to go then.